### ISSUE TITLE
rename .info() to .update() or .refresh()

### ISSUE THREAD

**Issue by gijzelaerr** [python-oca/python-oca#44]
hi! Is it an idea to rename .info() to .update() or .refresh()? It took me a while to figure out what was actually happening, and I think maybe those names make more sense? Maybe also an idea to call it implicitly on object instantiation? To me it looks like this is something you would always want to do.

**Comment by sysradium** [comment]
Yes, it looks weird but if I understand the author correctly that was done in such way just to replicate the OpenNebula RPC API. We may add a helper `refresh` functions for better understanding, however.

Or just improve docs a little bit.
